### ISSUE TITLE
Cover Block: Background Image is Repeated When Fixed Background setting is ON but Repeated Background is OFF

### ISSUE THREAD

**Issue by gwensmithx** [Automattic/wp-calypso#64682]
### Quick summary

I ran into an issue today where the Cover Block background image was repeating only when the Fixed Background option is toggled ON even though Repeat Background is toggled OFF. When the Fixed Background option is OFF then the image fills the space and does not repeat. As soon as the Fixed Background option is toggled on, the image repeats, regardless of whether the Repeat Background setting is on or off.

It seems to be with images that possibly are not wide enough to fill the screen to begin with, but then why does the image fill perfectly with both settings toggled off?

This happens on both Simple and AT sites - multiple themes.

Video:

https://user-images.githubusercontent.com/5779512/173929777-f2ba6181-9734-4d85-85f6-f673342861f5.mp4

 

### Steps to reproduce

1. Add a Cover Block to any page or post
2. Set to Full Width (this doesn't seem to  matter too much as I was able to reproduce on other sizes as well)
3. Add an image around 1200 wide or less as the background image
4. Notice it fills the space properly without any settings toggled on
5. Now toggle on Fixed Background but leave Repeated Background off
6. See the image repeats

### What you expected to happen

I expected that the image to look how it did without any setting toggled on (filling up the space - no repeating) but with the Fixed Background effect.

### What actually happened

When the Fixed Background option is toggled on the image shrinks and repeats itself regardless of whether the Repeated Background setting is on or off.

### Context

#35811576-hc / #5300222-zd

Chat / ticket

### Simple, Atomic or both?

Simple, Atomic

### Theme-specific issue?

No

### Browser, operating system and other notes

Saw the issue on Chrome, Firefox and Safari on MacOS. Also saw on Windows / Chrome using Browerstack.

### Reproducibility

Consistent

### Severity

All

### Available workarounds?

Yes, easy to implement

### Workaround details

The only workaround I can think of right now is to turn off the Fixed Background setting. There may be a CSS workaround but I haven't had time to test that out yet. They can also upload an image that is larger. Images that were at least 1800 w seemed to work as expected.

**Comment by filipanoscampos** [comment]
Ran into a case of this, here 5316934-zd-woothemes
Shared the following CSS for the user to try

```
/* Stop fixed Cover Block background image from repeating (FC) */
.wp-block-cover__image-background{
    background-size: cover;
 }
```

**Comment by aisajib** [comment]
📌 HOUSEKEEPING
- Labels ✅ 
- Priority ✅ 
- Replicable on Core - No

📌 SCRUBBING

- Tested on Simple ✅ 
- Tested on AT ✅ 
- Tested on Self-hosted ✅ 

📌 FINDINGS/SCREENSHOTS/VIDEO
I was able to replicate the issue on both simple and atomic sites. Wasn't able to reproduce it on a self-hosted site. 

![simple-fixed-background-issue](https://user-images.githubusercontent.com/2541844/175369720-29fafdd0-723c-4703-ba0b-a5b87813642f.png)



📌 ACTIONS
- Marked as Triaged for Quality Squad review

**Comment by dcoleonline** [comment]
5323845-zd

I applied this workaround CSS for now:

```
/* WP Support Workaround - Stop fixed Cover Block background image from repeating 35894103-hc */
.wp-block-cover__image-background{
    background-size: cover;
 }
```

Please notify them when the issue is resolved so that they can remove the above CSS.

**Comment by KirkwallDay** [comment]
- [ ] Found another case here: 5329121-zen

**Comment by jp-imagines** [comment]
Another report in 35938503-hc. User said this Cover block was previously working as expected, but suddenly changed after they published a new page on their site. 

- [ ] Follow-up: 5336381-zd-woothemes

**Comment by KirkwallDay** [comment]
Another instance here: 5325449-zen

**Comment by nagpai** [comment]
Another instance here too - 32995519-hc

Gave the CSS workaround.

**Comment by renata-franco** [comment]
Another one here: 5341159-zd-woothemes 
Provided the CSS workaround.

**Comment by mikeicode** [comment]
I've stopped using this effect completely for DIFM-L site builds because of this issue and have noticed it caused the repeating issue on previous sites I built.

TL;DR this issue affects many sites, anyone who has used this feature in the past (they won't all have noticed and reported the issue).

**Comment by mikeicode** [comment]
I ran into this using Hever pdEkSe-EW-p2

Used the following which is more specific than the CSS above, this targets only Cover blocks set to "Fixed" (parallax).

```
.wp-block-cover__image-background.has-parallax {
  background-size: cover;
}
```

**Comment by TeniCola** [comment]
Another report in 36062397-hc

Provided the CSS workaround.

**Comment by matiasbenedetto** [comment]
A PR with potential fix for this: https://github.com/WordPress/gutenberg/pull/42470

**Comment by jp-imagines** [comment]
Another instance in 34152764-hc.

- [ ] ZD for follow-up if needed: 5404763-zd-woothemes

**Comment by yansern** [comment]
While the actual fix is in Gutenberg https://github.com/WordPress/gutenberg/pull/42470, if escalation is needed, here's a phab patch for a temp fix in WPCOM. D84701-code

Also related: https://github.com/Automattic/wp-calypso/issues/57390

**Comment by rw-ye** [comment]
Another report:
 - [ ] 5391043-zd-woothemes

Provided CSS workaround.

**Comment by dolgelukkig** [comment]
Another one : 
 
- [ ] 35342093-hc

Shared CSS work around

**Comment by jordesign** [comment]
- [ ] Another user in 36206742-hc

I've provided the CSS workaround and opened a follow up ticket in 5426838-zd-woothemes

**Comment by ccwalburn** [comment]
- [ ] Another report 5425584-zen

Provided the CSS workaround.

**Comment by tellisbethel** [comment]
- [ ] Another report here: 5423518-zd-woothemes
Provided CSS workaround.

**Comment by mikeicode** [comment]
I ran into this again pdEkSe-Om-p2

**Comment by daledupreez** [comment]
The related fix in the upstream Gutenberg repo is scheduled for inclusion in Gutenberg 13.8, which should be released in the next week. We'll then need to get this rolled out to WordPress.com, but that should happen relatively soon after release.

**Comment by renata-franco** [comment]
Hi @daledupreez It's me again. Question: once Gutenberg 13.8 is released, it can take weeks to get to WordPress.com, or should be less time? In average, anything?

**Comment by daledupreez** [comment]
Hi @renata-franco, we're already tracking the work for Gutenberg 13.8 in https://github.com/Automattic/wp-calypso/issues/66094, so you should be able to track our progress there. I know that we had some blocking issues that delayed our rollout of Gutenberg 13.7, so Gutenberg 13.7.3 was only rolled out at the start of this week. That's also part of why I am not sure when the rollout will occur.

**Comment by daledupreez** [comment]
I am going to mark this issue as resolved. Gutenberg 13.8.0 was rolled out to Simple sites on 4 August, and to Atomic sites on 5 August.

**Comment by Greatdane** [comment]
Hi everyone 👋 Just a quick note to let you all know @Automattic/dotcom-triage-pa-group have finished following up with affected users reported in this thread.

Thanks for the reports, please let us know here if you see any future recurrence of the issue.
